massa-pool-exports/src/config.rs

```diff
@@ -13,6 +13,8 @@ pub struct PoolConfig {
     pub max_block_size: u32,
     /// maximal gas per block
     pub max_block_gas: u64,
+    /// Gas used by a base operation (transaction, roll buy, roll sell)
```

Comment:
it applies to all types of ops (not only rollbuy/rollsell/tx)

Reply:
changed the comment